scheme/tpm-enacttrust/test/cmd/gen-token/README.md

```diff
@@ -0,0 +1,43 @@
+## Token generation
+
+`gen-token` is used to generate test attestation tokens from JSON descriptions
+of the payload. This is build and used when running the tests.
+
+## Token Format
+
+An attestation token has the following format:
+
+    TPMS_ATTEST_LEN||TPMS_ATTEST||TPMS_ATTEST_SIGNATURE
+
+where
+
+`TPMS_ATTEST_LEN` is a uint16 values containing the length (in bytes) of the
+following `TPMS_ATTEST` structure. `TPMS_ATTEST` is structured according to
+[TPM 2.0
+Specification](https://trustedcomputinggroup.org/wp-content/uploads/TCG_TPM2_r1p59_Part2_Structures_pub.pdf).
```

Comment:
.nit: Is this new line intentional

Reply:
yes, this has been automatically formatted to adhere to 79 columns (the standard to README files).

Reply:
Thanks!